### PR TITLE
fix the global normalization in bias field correction

### PR DESCRIPTION
**Pull request by dkuegler**
To address incorrect etiv estimates, several changes to the bias field estimation and the pipeline are implemented
- use the white matter mask of the aseg to identify the rescaling slope so images have a predefined white matter intensity.
- add ifh2hdr from Freesurfer to the docker image (used by talairach_avi)
- change the directory the bashrc is written to in the docker image
- general cleanup and formatting of n4_bias_correct.py
- add overloads to image_io.py
- fix python-s reference when it does not exist (the idea is that in the docker, packages from user-space , so a mounted home directory, should never be used)

Addresses #378 .

### CODE REVIEW

File: run_fastsurfer.sh [hunk 1]
```diff
@@ -730,7 +734,7 @@ if [ "$run_seg_pipeline" == "1" ]
       then
         # this will always run, since norm_name is set to subject_dir/mri/orig_nu.mgz, if it is not passed/empty
         echo "INFO: Running N4 bias-field correction" | tee -a "$seg_log"
-        cmd="$python ${reconsurfdir}/N4_bias_correct.py --in $conformed_name --out $norm_name --mask $mask_name --threads $threads"
+        cmd="$python ${reconsurfdir}/N4_bias_correct.py --in $conformed_name --out $norm_name --mask $asegdkt_segfile --threads $threads"
```
Comment (m-reuter) [inline run_fastsurfer.sh]: Why was this changed to asegdkt instead of mask ?

Comment (m-reuter) [reply run_fastsurfer.sh]: maybe this should be the --aseg flag?

File: recon_surf/N4_bias_correct.py [hunk 2]
```diff
@@ -177,257 +184,388 @@ def N4correctITK(
...
 
-    if not options.skipwm:
-        print("\nnormalize WM to 110")
-        itkcorrected = normalizeWM(itkcorrected, itkmask, centroid=talorig)
+    target_wm = 110.
+
+    if options.skipwm:
+        logger.info("Skipping WM normalization, ignoring talairach and aseg inputs")
+        # normalize to average input intensity
+        m_bf_img = get_image_mean(itk_bfcorr_image)
```
Comment (m-reuter) [inline recon_surf/N4_bias_correct.py]: Maybe here we should get the means in the mask (if it is passed) as we do not care much about the intensities outside the brain.

**Comment by dkuegler** [comment]
@m-reuter The call to N4_bias_correct was in fact wrong.
I fixed that now. I tested the script not with the run_fastsurfer.sh script, but directly that script, so it should still be correct, it just means, that my tests/validation for the etiv values and the PV-effects are based on the wrong script...
using --mask was a version that I considered, but then I changed it again because it was potentially confusing and not clear.

So, unfortunate... It is going to delay any new release...

File: run_fastsurfer.sh (hunk 1 above)
Comment (dkuegler) [reply run_fastsurfer.sh]: yes, it should :(

File: recon_surf/N4_bias_correct.py (hunk 2 above)
Comment (dkuegler) [reply recon_surf/N4_bias_correct.py]: I think it probably makes little difference, but we can certainly do that...
